fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1936,7 +1936,14 @@ public void replayCreateTable(CreateTableInfo info) {
         Locker locker = new Locker();
         locker.lockDatabase(db.getId(), LockType.WRITE);
         try {
-            db.registerTableUnlocked(table);
+            if (!db.registerTableUnlocked(table)) {
+                if (table != null && !table.isTemporaryTable()) {
```

Comment:
It's not reasonable to remove the original table if create failed. 
As you said
`In replay create table, there must not be a table with the same name, unless there is a bug.`
We should fix the bug that causes the duplicate table names instead of ignoring it here.

Reply:
But after an error occurs, there needs to be a defense to recover the FE.